src/main/target/KAKUTEF4/target.c

```diff
@@ -27,13 +27,30 @@
 #include "drivers/timer_def.h"
 
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
+
+	#if defined(FLYWOOF405)
```

Comment:
This looks a lot better now, thanks.

One thing that is left is that the indentation is incorrect in this file. In the Betaflight source, please use 4 spaces instead of 1 tab, and change your editor accordingly.
Also, note that preprocessor directives are usually not indented.

Reply:
All right, I'll fix it right away~